contrib/tetragon-rthooks/cmd/oci-hook/main.go

```diff
@@ -327,8 +327,11 @@ func main() {
 
 	switch cliConf.HookName {
 	case "createContainer":
+		// do nothing
+		log.Info("nothing to do", "hook", "createContainer")
```

Comment:
Can we make it a Debug log?

Reply:
I'd like to keep it as it is, if that's OK. If everything's correct, we should never hit this. And if we do, it's one message per container, which I think is fine.

Reply:
ok, if we're expecting to never hit this then yeah it's fine :)